scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -1029,7 +1029,9 @@ def c1(x):
         assert_(np.all(res.x >= np.array(bounds)[:, 0]))
         assert_(np.all(res.x <= np.array(bounds)[:, 1]))
 
-    def test_L5(self):
+    @pytest.mark.parametrize("random_generator",
```

Comment:
probably best not to try this on one of the `test_Lx` series. They can take a long time to complete as it is.
The addition of the extra keywords should tested along the lines of `test_population_initiation` (L499-508).